src/models/messaging-channel.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const messageSchema = new Schema(
+  {
+    content: {
+      type: String,
+      required: true,
+    },
+    sender: {
+      type: Schema.Types.ObjectId,
+      ref: "Student",
+      required: true,
+    },
+    isRead: {
+      type: Boolean,
+      required: true,
+      default: false,
+    },
+  },
+  { timestamps: true }
+);
+const messagingChannnelSchema = new Schema(
+    {
+      contacts: {
```

Comment:
what does contact stand for here? are you referencing another object? if yes it needs a ref property